### PR TITLE
Fixed issue #19631: Very hard to match attributes in the CPDB when importing CSV data

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: assets/packages/adminbasics/src/components/panelsanimation.js [hunk 1]
```diff
@@ -10,7 +10,13 @@ export default function panelsAnimation() {
         /**
          * Card shown one by one
          */
-        document.querySelectorAll(".card").forEach(function (e, i) {
+        /** 
+         * @todo Added .welcome to restrict this to the welcome page (as it was intended), but the animation doesn't work
```
Comment (Shnoulle) [inline assets/packages/adminbasics/src/components/panelsanimation.js:14]: Maybe just remove it ?

No animation can be really better for usage . It's not a game here.

File: assets/packages/adminbasics/src/components/panelsanimation.js [hunk 2]
```diff
@@ -10,7 +10,13 @@ export default function panelsAnimation() {
...
+         *       anyway because the cards are already visible. On older versions, the panels (now cards) were hidden by
+         *       the definition of ".welcome .panel" in adminbasics.css. We should either fix all the CSS related to these
+         *       panels, or clean it and remove this CSS if the animation is not needed.
```
Comment (Shnoulle) [inline assets/packages/adminbasics/src/components/panelsanimation.js:17]: Animation not needed :+1:

File: assets/packages/adminbasics/src/components/panelsanimation.js (hunk 1 above)
Comment (gabrieljenik) [reply assets/packages/adminbasics/src/components/panelsanimation.js:14]: I would remove it on another ticket, focusing specially on that.
This is just for fixing the CPDB problem

**Comment by Shnoulle** [comment]
Some other dev like to fix a bunch of issue in one PR …

**Pull request merged**